src/renderer/codeMirror/index.js

```diff
@@ -1,6 +1,10 @@
 import { filter } from 'fuzzaldrin'
 import 'codemirror/addon/edit/closebrackets'
 import 'codemirror/addon/edit/closetag'
+import 'codemirror/addon/search/searchcursor'
+import 'codemirror/addon/dialog/dialog'
+import 'codemirror/addon/search/search'
+import 'codemirror/addon/search/jump-to-line'
```

Comment:
Is this needed for search or only for the `Alt+G` shortcut? Please remove the dependency if it's not necessary for seach as it may conflict with other functions (later).

Reply:
Only jump-to-line is for alt+g, the rest is required for search.  I expect in the long term that the code editor will get more 'native' search removing the dialog need.  As far as I know the addons should not cause any code conflict as they are siloed into their own class.

Reply:
Core problem that the feature is undocumented and it doesn't for for preview mode.

Reply:
Sure, that was largely why I was leaving it undocumented too, was it only worked in the code editor.  To confirm you just want jump-to-line removed correct?

Reply:
Yes, please.